testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -287,6 +290,316 @@ def test_write_none(self):
             w.write(None)
 
 
+class BaseTimestepTest(TestCase):
+    """Test all the base functionality of a Timestep
+
+    All Timesteps must pass these tests!
+
+    These test the Timestep independent of the Reader which it
+    comes into contact with.  Failures here are the Timesteps fault.
+    """
+    # define the class made in test
+    Timestep = mda.coordinates.base.Timestep
+    name = "base"  # for error messages only
+    size = 10  # size of arrays, 10 is enough to allow slicing etc
+    # each coord is unique
+    refpos = np.arange(size * 3, dtype=np.float32).reshape(size, 3)
```

Comment:
It may be nice to have reference positions that have decimal figures. It would allow to see if these decimal are read and write as expected. It could be done rather easily by multiplying the `refpos` array by 1.234 for instance so the decimal figures are different from each other.